.github/workflows/copy-md.yml

```diff
@@ -13,13 +16,13 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Set Up Node.js
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
-          node-version:
+          node-version: 16
 
       - name: Install Dependencies
         run: |
-          npm install -g jq   
+          yarn install -g jq   
```

Comment:
Why not installing using `apt install jq` instead of using npm?

Reply:
either way

Reply:
Seems like yarn had the same issue than npm: https://github.com/cosmos/cosmos-sdk-docs/actions/runs/6148633384/job/16682750925?pr=28, so hopefully apt will work.

Reply:
I think `sudo apt-get install jq -y` is better than `sudo apt install jq -y` looking at the log, as `apt install` isn't stable (TIL).

Reply:
jq is already there, https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md. no need for it, just found this

Reply:
lol. better :)